index.html

```diff
@@ -68,10 +68,10 @@
         synchronization to a playing audio or video media stream is needed,
         and makes recommendations for new or changed web APIs to realize these
         requirements. The goal is to extend the existing support in HTML for
-        text track cue events to add support for dynamic content replacement
-        cues and generic metadata events that drive synchronized interactive
-        media experiences, and improve the timing accuracy of rendering of web
-        content intended to be synchronized with audio or video media playback.
+        text track cues to add support for dynamic content replacement cues and
+        generic metadata cues that drive synchronized interactive media
```

Comment:
In my mental model, it would be better to use "data cues" instead of "metadata cues" to be more general.

Reply:
I agree, I have just pushed a commit to change this.